corehq/apps/hqadmin/tests/test_pillowtop.py

```diff
@@ -52,22 +55,21 @@ def test_with_blacklist_items(self):
         self.assertEqual(expected_pillows, set([c.name for c in configs_back]))
 
     def test_loading_existing_conf_file(self):
+        config = {'include_groups': ['mvp_indicators']}
 
-        expected_action = {'include_groups': ['mvp_indicators']}
-
-        action = get_single_pillow_action('staging')
-        self.assertEqual(action.to_json(), expected_action)
-
-    def test_loading_no_existing_conf_file(self):
-        action = get_single_pillow_action('foo')
-        self.assertIsNone(action)
+        action = get_single_pillow_action(config)
+        self.assertEqual(action.to_json(), config)
 
     def test_india_server_exclusions(self):
         self.pillowtops['fluff'] = [
             'custom.bihar.models.CareBiharFluffPillow',
             'custom.opm.models.OpmUserFluffPillow',
         ]
+        configs = [
+            {'exclude_groups': ['mvp_indicators', 'fluff']},
+            {'include_pillows': ['custom.bihar.models.CareBiharFluffPillow']},
+        ]
 
-        pillows = [c.name for c in get_pillows_for_env('softlayer', self.pillowtops)]
+        pillows = [c.name for c in get_pillows_for_env(configs, self.pillowtops)]
         self.assertNotIn('OpmUserFluffPillow', pillows)
         self.assertIn('CareBiharFluffPillow', pillows)
```

Comment:
@czue this makes the pillowtop tests work and passes in the pillow config to the management command. it goes along with this commit: https://github.com/dimagi/commcare-hq-deploy/commit/d04b9a68b177e14757ee11f0dc8d80fb70eded13. i kind of screwed up the diff with the `pillowtop_settings.py`. essentially i moved it from fab to here and made it use a dict that was passed in instead of reading it from the services dir.

we still have a dep on the fab dir though. the supervisor command reads the supervisor templates from it so we need to keep this as a submodule. since we already have to have this as a submodule we could just read the pillow config yml files from that submodule if we think passing the config as a param is too clunky.